app/views/hosts/_selected_hosts.html.erb

```diff
@@ -1,4 +1,6 @@
 <div class="row">
+  <%= alert(:class => 'alert-info hide', :close => true, :header => '',
```

Comment:
@ohadlevy 

I actually like the way it's address in the gif. I don't think it should be a warning as nothing "wrong" happened.